src/golang/config/config.go

```diff
@@ -17,11 +17,15 @@ var (
 )
 
 type serverConfiguration struct {
-	AqPath             string                `yaml:"aqPath"`
-	EncryptionKey      string                `yaml:"encryptionKey"`
-	RetentionJobPeriod string                `yaml:"retentionJobPeriod"`
-	ApiKey             string                `yaml:"apiKey"`
-	StorageConfig      *shared.StorageConfig `yaml:"storageConfig"`
+	AqPath             string                `yaml:"aqPath" json:"aqPath"`
+	EncryptionKey      string                `yaml:"encryptionKey" json:"encryptionKey"`
```

Comment:
I've added json keys here, but realize that we probably don't want to expose the encryption key. I'm probably going to make a struct in my handler to contain all fields except for the encryptionKey.

Reply:
Going to remove these json tags, no longer needed here.